starter-code/styles/modules.css

```diff
@@ -1,57 +1,63 @@
 .site-header {
   background: #FFEAAA;
 }
-
-/* REVIEW: Float the main header to the left, to make room for the hamburger menu. */
 .heading-group-h1 {
   float: left;
   padding: 0 0 .2em 1em;
 }
-
-/* TODO: Add styles for the icon-menu class. It should be bigger, and off to the right. */
-
 .main-nav {
   padding: 0.1em;
 }
-
 .main-nav ul {
   display: none;
   padding: 0;
   list-style: none;
   clear: both;
 }
-
 .main-nav:hover ul {
   display: block;
 }
-
 .main-nav li {
   margin-right: .4em;
   font-size: 1.5em;
   padding: .5em .2em;
   border-bottom: 1px solid #ccc;
 }
-
 #filters {
   padding: 0;
   list-style: none;
 }
-
 .template {
   display: none;
 }
-
 .byline {
   color: #777;
 }
-
 .byline a {
   font-weight: bold;
   text-decoration: none;
 }
-
-/* TODO: Now add styles for how things should look on a larger screen:
-         - Hide the hamburger.
-         - Main should be all on one row, with padding spaced better for a larger screen.
-         - Just, ya know, try to make it look good! */
-/* @media only screen... */
+.icon-menu:before {
+float: right;
+font-size: 3em;
+margin-top: .4em;
+}
+@media (min-width: 800px) {
+  .icon-menu:before {
+    display: none;
+  }
+}
+@media (min-width: 800px) {
+  .main-nav ul {
+    display: inherit;
+    text-align: center;
+    margin-left: -60px;
+  }
+}
+@media (min-width: 800px) {
+  .main-nav li {
+      display:inline;
+      margin-left: 2em;
+      margin-right: 2em;
+  }
```

Comment:
Great job on the media queries! One thing - since they are all the same 'min-width: 800px' you don't need three media queries. They can all go under just one for cleaner code.